pyccel/commands/epyccel.py

```diff
@@ -356,10 +439,10 @@ def epyccel( python_function_or_module, **kwargs ):
         # Master process calls epyccel
         if comm.rank == root:
             try:
-                mod, fun = epyccel_seq( python_function_or_module, **kwargs )
+                mod, fun = epyccel_seq( python_function_class_or_module, **kwargs )
```

Comment:
Here I could call the variable `obj` rather than `fun`, as it may also be a class.

Reply:
Fixed e57c25121